### ISSUE TITLE
Hacktobefest '20

### ISSUE THREAD

**Issue by achaljhawar** [achaljhawar/DevFolio#9]
This issue is meant to be a good first issue to get into this repository, and into open source contribution overall, as it will require to touch few existing files and is documented extensively. If this is your first contribution to open source, I encourage you to try this issue. Please note that I will only accept one of the good first issues per person to give as many people as possible the opportunity for a first contribution.

Please read our [contribution guidelines](CONTRIBUTING.MD) and [Code of Conduct](CODE_OF_CONDUCT.md) before picking this up, it will also lead you through the process of setting up the repository.

**Comment by keivalya** [comment]
Yepp!
Interested

**Comment by achaljhawar** [comment]
So @keivalya you can start by joining the <a href="https://discord.gg/uDdH2p5">Discord server</a>.Where you can suggest things and make PRs to DevFolio.

**Comment by achaljhawar** [comment]
You can start by registering at <a href="https://hacktoberfest.digitalocean.com/login">Hacktoberfest's site</a>with your Github account.

**Comment by keivalya** [comment]
DONE!!

**Comment by Yoddha99** [comment]
Can i also work on this?

**Comment by achaljhawar** [comment]
Yes, every contributor is welcome @Yoddha99.You can join the <a href="https://discord.gg/uDdH2p5">discord server</a>.And start contributing!
